lib/utils.js

```diff
@@ -13,3 +13,21 @@ export function isInput(element: HTMLElement): boolean {
 
   return tagName === 'INPUT' || tagName === 'TEXTAREA';
 }
+
+/**
+ * Maps [keycodes](https://www.npmjs.com/package/keycodes) key names to KeyboardEvent.key values.
+ */
+export function keyNameVals(keyName: ?string): string {
```

Comment:
In lines of the functions: `keyCodes` and `keyNames` I would call this `keyValues`, what do you think?